src/scroll_list/ScrollList.js

```diff
@@ -379,6 +380,17 @@ define(function(require) {
          */
         this.onScrollToItemStarted = Observable.newObservable();
 
+        /**
+         * Observable for subscribing to scroll events beyond AwesomeMap boundaries.
+         *
+         * @method AwesomeMap#onScrollPastTopBoundary
+         * @param {Function} callback
+         *        Invoked with (sender, {
+         *            boundary: {@link BoundaryTypes}
+         *        })
+         */
+        this.onScrollPastBoundary = Observable.newObservable();
```

Comment:
I discussed this very question with a couple people, and we mostly landed on one observable vs multiple.  I think there's some nice simplicity in having different observables for each, but that would also cost alot of code duplication.  And it would be fairly trivial to `switch(){}` off of boundary types in the subscriber logic, if they only cared about a couple different ones. I dunno.